cea/optimization/preprocessing/decentralized_buildings.py

```diff
@@ -31,7 +34,10 @@ def decentralized_main(locator, building_names, gv):
     :rtype: Nonetype
     """
     t0 = time.clock()
-    geothermal_potential = pd.read_csv(locator.get_geothermal_potential(), index_col="Name")
+    prop_geometry = Gdf.from_file(locator.get_zone_geometry())
+    geothermal_potential_data = dbfreader.dbf_to_dataframe(locator.get_building_supply())
+    geothermal_potential = geothermal_potential_data[['Name', 'restr_geo']]
+    geothermal_potential['Area_geo'] = geothermal_potential['restr_geo'] * prop_geometry.area
```

Comment:
This will be a source of error, better use merge on=Name. Otherwise it might happen that the order of the cells be shifted.